index.html

```diff
@@ -0,0 +1,117 @@
+                        <li><a href="https://www.linkedin.com/in/gerson-sanchez-88309b57/"><img
+                                    src="img\Linkedin icon.svg" alt="logo linkedin"></a></li>
+                        <li><a href="https://angel.co/"><img src="img\angel list.svg" alt="logo angel list"></a></li>
+                        <li><a href="https://twitter.com/gealsanchez"><img src="img\twitter.svg" alt="logo twitter"></a></li>
+                        <li> <a href="https://medium.com/tag/javascript"><img src="img\medium.svg" alt="logo medium"></a></li>
+                    </ul>
+                </div>
+
+                <div class="desktopmenu d-flex justify-content-between">
+                    <a class="ms-5 ps-5 logo" href=""><img src="img/logo.png" alt="logo creative commons"></a>
+                    <ul>
+                        <a style="color: rgb(255, 51, 0);" href="about.html">
+                            <li>About</li>
+                        </a>
+                        <a href="#">
+                            <li class="ms-5">Program</li>
+                        </a>
+                        <a href="#">
+                            <li class="ms-5">Join</li>
+                        </a>
+                        <a href="#">
+                            <li class="ms-5">Sponsor</li>
+                        </a>
+                        <a href="#">
+                            <li class="ms-5 me-3">News</li>
+                        </a>
+                        <a style="border: 3px solid rgb(255, 51, 0); border-radius: 5px;color: rgb(255, 51, 0);"
+                            href="#">
+                            <li class="mx-5">Campaign</li>
+                        </a>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+
+        <div class="container text-dark my-2 pb-5 sm-fs-4">
+            <h3 class="pt-5 pb-2 mt-1" style="color: rgb(255, 51, 0); font-weight: 400; font-size: 32px;">"Hello, Sharing world"</h3>
+            <h1 style="color: rgb(255, 51, 0);">CONTAINER FLUID</h1>
+            <h1 style="color: rgb(255, 51, 0);">ARTS FESTIVAL</h1>
+            <h1 style="color: rgb(255, 51, 0);">2022</h1>
+            <p class="bg-light my-5 p-1">Container Festival is without doubt the Bucaramanga's most famous music and
+                performing arts festival, if not the
+                world's. Embracing every genre across the spectrum </p>
+            <h1 class="date">2022.11.03(THU) ~ 06(SAT)</h1>
+            <small class="pb-5">@National Museum, Modern Art Museum and more</small>
+        </div>
+    </div>
+
+    <div class="programwrapper">
+        <h1 class="text-center fs-3 " style="color: #fff;">Main Program</h1>
+        <hr class='mb-5'
+            style="color:rgb(255, 51, 0,1) ;border: 2px solid rgb(255, 51, 0,1);width: 25%; margin-left: 37%;">
```

Comment:
[OPTIONAL]

- [ ] It's better if you style the elements in a separate CSS file instead of inline styles to make maintainability and readability better 👍